<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QDialog" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Trace</string>
  </property>
  <property name="styleSheet">
   <string notr="true">backdrop-filter: blur(10px);</string>
  </property>
  <widget class="QWidget" name="registration_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>23989</width>
     <height>2349023</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	height: 520px;
	width: 400px;
	background-color: rgb(40, 39, 46);
	transform: translate(-50%,-50%);
	position: absolute;
}</string>
   </property>
   <widget class="QLineEdit" name="login_input_field">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>110</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(55, 54, 61);
}

QLineEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input_field">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>190</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(55, 54, 61);
}

QLineEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel 
{
	font-family: 'Poppins',sans-serif;
	letter-spacing: 0.5px;
	color: #ffffff;
	font-weight: 500;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="password_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel 
{
	font-family: 'Poppins',sans-serif;
	letter-spacing: 0.5px;
	color: #ffffff;
	font-weight: 500;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_up_button">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>330</y>
      <width>321</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: #ffffff;
	color: #080710;
	font-size: 18px;
	font-family: 'Poppins',sans-serif;
	letter-spacing: 0.5px;
	border-radius: 20px;
	font-weight: 600;
}
QPushButton:hover {
	border: 2px solid rgb(40, 39, 46);
}</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="dont_have_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	color: rgb(155, 155, 158);
	font-family: 'Poppins',sans-serif;
	letter-spacing: 0.5px;
	font-weight: 500;
	font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string>Have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log_in_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: #ffffff;
	font-family: 'Poppins',sans-serif;
	letter-spacing: 0.5px;
	color: #ffffff;
	font-weight: 500;
	font-size: 13px;
	background-color: rgb(40, 39, 46);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QLabel" name="create_account_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>231</width>
      <height>38</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	font-family: 'Poppins',sans-serif;
	color: #ffffff;
  	letter-spacing: 0.5px;
	font-size: 32px;
  	font-weight: 500;
  	line-height: 42px;
  	text-align: center;
}</string>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="confirm_password_input_field">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>270</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 2px solid rgb(37, 39, 48);
	border-radius: 20px;
	color: #FFF;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(55, 54, 61);
}

QLineEdit:hover {
	border: 2px solid rgb(48, 50, 62);
}

QLineEdit:focus {
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Repeat your password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel 
{
	font-family: 'Poppins',sans-serif;
	letter-spacing: 0.5px;
	color: #ffffff;
	font-weight: 500;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string>Confirm password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
